src/lifecycle/created.js

```diff
@@ -55,7 +55,7 @@ export default function (opts) {
 
   return function () {
     let info = data(this, opts.id);
-    let isNative = this.createdCallback;
+    let isNative = !this.hasOwnProperty('_isPolyfill') && this.createdCallback || !(this._isPolyfill = true);
```

Comment:
> This is equivalent to:

Yep, but if we can pre-calculate then we don't have to do it multiple times.

Also, if you're going to store information on an element, probably better to use `util/data`.